langchain/package.json

```diff
@@ -1512,7 +1512,7 @@
     "js-tiktoken": "^1.0.7",
     "js-yaml": "^4.1.0",
     "jsonpointer": "^5.0.1",
-    "langchainhub": "~0.0.6",
+    "langchainhub": "^0.0.8",
```

Comment:
Should be tilde to allow for patch updates

Also, if a new version has been published, new installs will get that one?